pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -75,14 +76,28 @@ type Manager struct {
 }
 
 // NewManager returns a new node shutdown manager.
-func NewManager(getPodsFunc eviction.ActivePodsFunc, killPodFunc eviction.KillPodFunc, syncNodeStatus func(), shutdownGracePeriodRequested, shutdownGracePeriodCriticalPods time.Duration) (*Manager, lifecycle.PodAdmitHandler) {
+func NewManager(getPodsFunc eviction.ActivePodsFunc, killPodFunc eviction.KillPodFunc, syncNodeStatus func(),
+	shutdownGracePeriodRequested, shutdownGracePeriodCriticalPods time.Duration,
+	podPriorityShutdownGracePeriod []kubeletconfig.PodPriorityShutdownGracePeriod,
+) (*Manager, lifecycle.PodAdmitHandler) {
+
+	// Migration from the original configures
+	if !utilfeature.DefaultFeatureGate.Enabled(features.PodPriorityBasedGracefulShutdown) ||
```

Comment:
ping on this?

Reply:
re-opening this thread -

should we also check that GracefulNodeShutdown feature gate is enabled before we call migrateConfig?

Reply:
resolving, this is not needed anymore since due to above check of

```
if !utilfeature.DefaultFeatureGate.Enabled(features.GracefulNodeShutdown) {
		m := managerStub{}
		return m, m
	}
```